docs/integrations/creating-nodes/build/reference/ui-elements.md

```diff
@@ -564,4 +564,95 @@ The HTML editor allows users to create HTML templates in their workflows. The ed
 
 Refer to [`Html.node.ts`](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/Html/Html.node.ts){:target=_blank .external-link} for a live example.
 
+## Resource mapper
 
+If your node performs insert, update, or upsert operations, you need to send data from the node in a format supported by the service you're integrating with. A common pattern is to use a Set node before the node that sends data, to convert the data to match the schema of the service you're connecting to. The resource mapper UI component provides a way to get data into the required format directly within the node, rather than using a Set node.
+
+!!! note "Mapping and matching"
+	Matching is the process of using column names to identify the row(s) to update.
+	Mapping is the process of setting the input data to use as values when updating row(s).
+
+
+```js
+{
+	displayName: 'Columns',
+	name: 'columns', // The name used to reference the element UI within the code
+	type: 'resourceMapper', // The UI element type
+	default: {
```

Comment:
Can you also add the `value: null` should be used in `default`(as mentioned [here](https://www.notion.so/n8n/Resource-Mapper-Component-792800eddab54aa59d061039490eb50b?pvs=4#7bc84e3835974bc8bf95630c7c10068c))?